src/web/app/components/comment-box/comment-edit-form/comment-edit-form.component.html

```diff
@@ -33,7 +33,7 @@
     <tbody>
     <ng-container *ngFor="let commentVisibilityType of CommentVisibilityType | enumToArray">
       <tr *ngIf="visibilityStateMachine.isVisibilityTypeApplicable(commentVisibilityType)">
-        <td container="body"> <span [ngbTooltip]="commentVisibilityType | commentVisibilityTypeDescription">{{ commentVisibilityType | commentVisibilityTypeName }}</span></td>
+        <td><span container="body" [ngbTooltip]="commentVisibilityType | commentVisibilityTypeDescription">{{ commentVisibilityType | commentVisibilityTypeName }}</span></td>
```

Comment:
Does this need the `ngb-tooltip-class` class?

Reply:
Yup it needs. Thanks for catching it.